### ISSUE TITLE
Not support the extended interface

### ISSUE THREAD

**Issue by mato533** [fossamagna/google-script-dts-generator#193]
# About the issue
The extends statement is not taken into account when defining a function that uses an extended interface.
Interfaces in extend statements and extends statements itself are not printed.

# Example
## Source Code

```ts
interface IParams1 {
  attr1: string;
  attr2: number;
}
interface IParams2 extends IParams1 {
  attr3: boolean;
  attr4: {
    attr41: string;
  };
}
global.funcTest = (params: IParams2) => {
  const test = "hello " + params.attr1;
  console.log(test);
};
```

## Output d.ts file
```ts
declare namespace google {
    /**
     * Methods available to Google Apps Script
     */
    namespace script {
        interface IRun {
            funcTest(params: IParams2): void;
            // ... omit ...
        }

        interface IParams2 { // "extends IParams1" and "interface IParams1 ~"  are not printed 
            attr3: boolean;
            attr4: {
                attr41: string;
            };
        }

```

**Comment by fossamagna** [comment]
@mato533  I published [google-script-dts-generator@1.3.0](https://www.npmjs.com/package/google-script-dts-generator/v/1.3.0). check it please.

**Comment by mato533** [comment]
@fossamagna 
I checked that the problem was fixed!! Thx!!
